README.md

```diff
@@ -1,3 +1,52 @@
-# devprocess
+# Project Title
```

Comment:
I think this actually should be "DevProcess". The other sections are ok because we don't have defined them yet, but the title should be the repo name.